drivers/bluetooth/hci/h4.c

```diff
@@ -25,6 +25,7 @@
 #define BT_DBG_ENABLED IS_ENABLED(CONFIG_BT_DEBUG_HCI_DRIVER)
 #define LOG_MODULE_NAME bt_driver
 #include "common/log.h"
+#include "common/bt_str.h"
```

Comment:
can't comment on the commit msg, but you forgot to update the names (bt_to_str -> bt_str) there.